governance/votes/Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Template - [Executive Vote] BlockTower Andromeda Upgrade, Smart Burn Engine Deployment, Keeper Job Updates, Scope Defined Parameter Changes, Delegate Compensation, Ecosystem Actor Funding, Spark Protocol Proxy Spell Execution - July 12, 2023
```

Comment:
Seems to be missing the CRVV1ETHSTETH-A 1st Stage Offboarding. Please confirm if it's not meant for this exec. Would also reword to "Ecosystem Actor and Core Unit Funding Updates" to catch the yanked streams and MKR vesting to CUs. Applies to summary below.

Reply:
CRVV1ETHSTETH-A 1st Stage Offboarding falls under Scope Defined Parameter Changes for the purpose of the title. Agree on the Core Unit bit